src/blocks-transactions-loader/blocks-transactions-loader.ts

```diff
@@ -127,7 +127,7 @@ export class BlocksTransactionsLoader {
         {
           // We increased this from the default of 5000 because the transactions were
           // timing out and failing to upsert blocks
-          timeout: 30000,
+          timeout: 40000,
```

Comment:
Want to add an env var here so that we can scale back down without PR?

Reply:
Yeah good idea